testcases/kernel/syscalls/lgetxattr/lgetxattr02.c

```diff
@@ -67,7 +82,7 @@ static void verify_lgetxattr(unsigned int n)
 static void setup(void)
 {
 	int res;
-
+	SAFE_MKDIR(tmpdir, 0644);
```

Comment:
We are creating foler tmdir but we are not going into this folder. So testfile and symlink is not created in tmpdir. Right?

Reply:
Vamsi I don't see this code anymore. Did you remove it? How did you apply workaround to use roof file system? I cannot see the code change for that.